pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.pytest.ini_options]
-addopts = "--cov=kedro_mlflow --cov-report=html tests/"
```

Comment:
Why isn't it necessary anymore?

Reply:
ah my bad, I had removed this to be able to call `pytest ..path..to.test` manually, the `tests/` at the end blocked that from being possible

Reply:
(just for the record, I guess you're searching ``pytest -k "regex"`` to execute only some tests based on a regex)